### ISSUE TITLE
Import not working with proxy setting

### ISSUE THREAD

**Issue by Arrow-Li** [cn-poe-community/poexport#2]
Issue:
When use a proxy server for pob (update needed), import build through local export is not working.
![XXXX](https://user-images.githubusercontent.com/17962902/214789315-a774f3a2-ca25-4140-ba73-6db0130a2bf1.png)
---------
Setting:
![QQQQQQ](https://user-images.githubusercontent.com/17962902/214789534-98c33f4a-f1f5-4061-89c8-01ed87e1ecc2.png)
---------
Fix:
Mod lua script at `./Launch.lua`
```lua
        --- Line:299
	local proxy_url = self.proxyURL --- Add
	if string.find(url, "^http://localhost") == 1 then --- Add
		proxy_url = "" --- Add
	end --- Add

	local id = LaunchSubScript(script, "", "ConPrintf", url, params.header, params.body, self.connectionProtocol, proxy_url) --- Mod                                                 
```
![xxxxxxxxxx](https://user-images.githubusercontent.com/17962902/214791519-9c9b1d57-788e-4102-bdc6-117995a05faa.png)

PS: I'm not familiar with js, so just put patch code above.

**Comment by me1ting** [comment]
Your patch not works on my computer, but makes the POB crashed. I test it using socks5 proxy and http proxy, both of them crashed.

It's not a good way to change POB files to fix this question even if your patch works well, beacase POB updates may break the fix and bring other questions.

I made such tests:

- POB->socks5, fail
- POB->privoxy(http to socks5)->socks5, fail
- POB->privoxy(http to socks5) with updated settings ->socks5, success
- POB->my http2socks tool->socks5, success

I suggest using privoxy to privide http proxy. Here is my settings:
```
listen-address 127.0.0.1:1081
forward-socks5 / 127.0.0.1:1080 .
forward localhost/ .
permit-access 127.0.0.0/8
show-on-task-bar 1
close-button-minimizes 0
log-messages 1
```

**Comment by Arrow-Li** [comment]
![xxxx](https://user-images.githubusercontent.com/17962902/214820619-ff702a2c-83f4-4944-9d31-f8a49f12e370.png)



I test this patch works on my computer, perhaps post your patched code and crash message? 



--------
Because your tool is also patch pob file (change the tencent host name), I did same way.
Use others proxy software is not a good solution, Because proxy works will with any other softwares include pob itself.

**Comment by me1ting** [comment]
I retest this path , it works well.

Changing the tencent host name is the min way to make this tool working and probably has no conflict with pob updates in the future. 
But change the code of pob may bring errors when pob updated.

I will add support of this request as a experimental function.User should use it manually.

**Comment by me1ting** [comment]
Closed by 6de698d
